src/main/deploy/shooterData.csv

```diff
@@ -1,18 +1,14 @@
-Distance (inches),Angle (degrees),Velocity (RPM)
```

Comment:
We should be careful about merging this file because it contains older data points than #140.